src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -131,13 +121,27 @@ abstract contract IntegrationDeployer is Test, IUserDeployer {
     }
 
     function setUp() public virtual {
+        string memory deploymentInfoPath;
+        if (block.chainid == 1) {
+            _setUpMainnet();
+        } else if (block.chainid == 5) {
+            deploymentInfoPath = "script/utils/Goerli_current_deploy.config.json";
+            _setUpGoerli(deploymentInfoPath);
+        } else if (block.chainid == 31337) {
+            _setUpLocal();
+        } else {
+            revert();
```

Comment:
Probably good to have a message here, otherwise this would be a meganightmare to debug